src/service/viewer-impl.js

```diff
@@ -263,6 +273,14 @@ export class Viewer {
         || this.isWebviewEmbedded_
         || !ampdoc.isSingleDoc());
 
+    /**
+     * Whether the AMP document is embedded in a Chrome Custom Tab.
+     * @private @const {boolean}
+     */
+    const queryParams = parseQueryString(this.win.location.search);
```

Comment:
Move this above the comment. The JSDoc should be right above the ivar `isCctEmbedded_`.

Reply:
Oops. Fixed.